code/modules/mob/living/say.dm

```diff
@@ -204,6 +204,13 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 		spans |= SPAN_SINGING
 		message = "[randomnote] [message] [randomnote]"
 
+	///List of symbols that we dont want a dot after
+	var/list/punctuation = list("!","?",".")
+	///Last charachter in the message

```

Comment:
Truly ironic a PR about enforcing grammatical rules has a spelling mistake in it.